src/metabase/api/routes.clj

```diff
@@ -61,6 +62,8 @@
   (context "/email"           [] (+auth email/routes))
   (context "/embed"           [] (+message-only-exceptions embed/routes))
   (context "/field"           [] (+auth field/routes))
+  ;; TODO - fingerprint and comparison should be split out?
+  (context "/fingerprint"     [] (+auth fingerprint/routes))
```

Comment:
can we please call this "xray" or something? Having two different things named fingerprints is getting confusing